src/ContentHubClient.php

```diff
@@ -450,29 +450,34 @@ public function postEntities(CDFObject ...$objects) {
    *
    * @param string $uuid
    *   Entity UUID.
+   * @param bool $propagate
+   *   Entity propagation flag.
    *
    * @return \Psr\Http\Message\ResponseInterface
    *   Response.
    *
    * @throws \GuzzleHttp\Exception\RequestException
    */
-  public function deleteEntity($uuid) {
-    return $this->delete("entities/$uuid");
+  public function deleteEntity($uuid, $propagate = FALSE) {
+    return $this->delete("entities/$uuid", ['propagate' => $propagate]);
   }
 
   /**
    * Deletes multiple entity uuids.
    *
    * @param array $uuids
    *   Uuids to delete.
+   * @param bool $propagate
+   *   Propagation flag for Uuids.
```

Comment:
Let's keep it similar, as we have it for `deleteEntity()`
```suggestion
   *   Propagation flag for entities.
```

Reply:
Done